examples/basic_website_jsx/Makefile

```diff
@@ -0,0 +1,8 @@
+site_gen := make_site
+
+all:
+	ocamlfind ocamlc site_html.ml -package tyxml.ppx -short-paths -linkpkg -o ${site_gen}
```

Comment:
This makefile doesn't look correct. It should be site_html.re. And tyxml.jsx instead of tyxml.ppx?

Maybe the makefile should just call dune? It would be easier than adapting everything to support the reason syntax.